timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -26,20 +27,22 @@
 import com.palantir.atlasdb.util.AtlasDbMetrics;
 import com.palantir.remoting3.servers.jersey.HttpRemotingJerseyFeature;
 import com.palantir.timelock.paxos.TimeLockAgent;
-import com.palantir.tritium.metrics.MetricRegistries;
 
 import io.dropwizard.Application;
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
 
+/**
+ * Provides a way of launching an embedded TimeLock server using Dropwizard. Should only be used in tests.
+ */
 public class TimeLockServerLauncher extends Application<TimeLockServerConfiguration> {
     public static void main(String[] args) throws Exception {
         new TimeLockServerLauncher().run(args);
     }
 
     @Override
     public void initialize(Bootstrap<TimeLockServerConfiguration> bootstrap) {
-        MetricRegistry metricRegistry = MetricRegistries.createWithHdrHistogramReservoirs();
+        MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate("AtlasDbTest");
```

Comment:
Hmm. We might need a way to ensure that they are created differently - if you see `TestableTimelockCluster`, that creates multiple timelock servers in the same JVM, so this will throw when creating the second node.

`SharedMetricRegistries.getOrCreate(UUID.randomUUID().toString())` seems horrible, though